scripts/data/online_evals_generate_test_data.py

```diff
@@ -0,0 +1,164 @@
+#This file generates test data for the periodically_get_spans.py script
+
+import os
+from getpass import getpass
+
+import pandas as pd
+import phoenix as px
+
+from phoenix.session.evaluation import get_qa_with_reference, get_retrieved_documents
+from phoenix.trace import DocumentEvaluations, SpanEvaluations
+from phoenix.trace.langchain import LangChainInstrumentor
+from phoenix.experimental.evals import (
+    HallucinationEvaluator,
+    OpenAIModel,
+    QAEvaluator,
+    RelevanceEvaluator,
+    run_evals,
+)
+# LangChain imports
+from langchain.chains import RetrievalQA
+from langchain_community.document_loaders import GitbookLoader
+from langchain_community.embeddings import OpenAIEmbeddings
+from langchain.callbacks import StdOutCallbackHandler
+from langchain_openai import ChatOpenAI
+from langchain_community.vectorstores import Qdrant
+from phoenix.trace import TraceDataset
+
+def create_session():
+    session = px.launch_app()
+    return session
+
+def lookup_traces(session, selected_date):
+    # Get traces into a dataframe
+    spans_df = px.Client(endpoint=str(session.url)).get_spans_dataframe("span_kind == 'RETRIEVER'")
+
+    trace_df = px.Client(endpoint=str(session.url)).get_trace_dataset()
```

Comment:
you only need client. Also the session URL is technically not necessarily the endpoint. Notably in sagemaker notebooks. Why is this param needed